### ISSUE TITLE
Rotate pages for easy writing/drawing.

### ISSUE THREAD

**Issue by aero31aero** [xournalpp/xournalpp#4230]
**Is your feature request related to a problem? Please describe.**

I'm unable to write completely horizontal and even when I have a physical notebook to write on, I turn it something like 30degrees counter-clockwise in order to write horizontally in it.

Being unable to do that in Xournal++ keeps it to a hobby app for me instead of an actually productive one.

**Describe the solution you'd like**

Something like Krita's canvas rotating system:

![](https://i.imgur.com/m9RczOw.png)

Overall, Krita's canvas controls are way more fluid, so things like "hold X and drag to zoom in/out" etc make much more sense, but the rotation ability feels entirely absent right now.

**Describe alternatives you've considered**

Using Krita as a note taking program and having to save multiple individual documents, one for each page. That arrangement is very clunky.

**Comment by hyperupcall** [comment]
@aero31aero I think some sort of rotation would be useful - I'm not sure how feasible it is, I'll have to look into it

Can you describe your use case a bit more? You mention saving an individual document for each page - I assume these aren't pages if a book or slideshow presentation, but something else?

**Comment by michimussato** [comment]
I agree that rotating a page as demonstrated using Krita is usefol for a more ergonomic and precise way of interaction when handwriting (using a graphics tablet). I guess we do this all the time (maybe not even consciously) when writing/drawing on real paper.

**Comment by aero31aero** [comment]
Hey, I just wanna bring to your attention that this and the other issue you closed as duplicates are actually duplicates of each other but not the original issue, well, not exactly.

That issue is about the pages being rotated incorrectly in the pdf document itself, and could potentially be fixed by some pdf editing tool to correct the orientation. Also, a fix for that issue would probably be including 0,90,180,270 as rotation options, as well as potentially writing back the rotated page to the file.

This one is merely about how it's easier for people to tilt their pages when writing, so the desired solution would be rotation of the canvas by 0,10,20,30... degrees etc, and crucially, not writing these rotations to the files.
